<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-06-07T13:10:48 IST" name="Suite" finished-at="2021-06-07T13:11:02 IST" duration-ms="14212">
    <groups>
    </groups>
    <test started-at="2021-06-07T13:10:48 IST" name="Test" finished-at="2021-06-07T13:11:02 IST" duration-ms="14212">
      <class name="Base.BaseClass">
        <test-method is-config="true" signature="setup()[pri:0, instance:Base.BaseClass@31d7b7bf]" started-at="2021-06-07T13:10:48 IST" name="setup" finished-at="2021-06-07T13:10:53 IST" duration-ms="5834" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="startTest()[pri:0, instance:Base.BaseClass@31d7b7bf]" started-at="2021-06-07T13:10:53 IST" name="startTest" finished-at="2021-06-07T13:10:53 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startTest -->
        <test-method signature="firstTestCase()[pri:0, instance:Base.BaseClass@31d7b7bf]" started-at="2021-06-07T13:10:53 IST" name="firstTestCase" finished-at="2021-06-07T13:11:01 IST" duration-ms="8032" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Sample Test3 Page] but found [Sample Test Page]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Sample Test3 Page] but found [Sample Test Page]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Base.BaseClass.firstTestCase(BaseClass.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- firstTestCase -->
        <test-method signature="secondTestCase()[pri:1, instance:Base.BaseClass@31d7b7bf]" started-at="2021-06-07T13:11:01 IST" name="secondTestCase" finished-at="2021-06-07T13:11:02 IST" duration-ms="285" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- secondTestCase -->
        <test-method signature="ThirdTestCase()[pri:2, instance:Base.BaseClass@31d7b7bf]" started-at="2021-06-07T13:11:02 IST" name="ThirdTestCase" finished-at="2021-06-07T13:11:02 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ThirdTestCase -->
      </class> <!-- Base.BaseClass -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
